kodilla-testing/src/main/java/com/kodilla/testing/collection/OddNumbersExterminator.java

```diff
@@ -0,0 +1,31 @@
+package com.kodilla.testing.collection;
+
+import java.util.ArrayList;
+import java.lang.*;
+
+
+public class OddNumbersExterminator {
+    ArrayList<Integer> listEven;
```

Comment:
why do you introduce instance variable listEven? It will be enough to use variable which scope will be just 'exteriminate' method. What will happen with listEven when you call 'exterminate' twice?